charts/cinder-csi-plugin/templates/_helpers.tpl

```diff
@@ -62,6 +62,9 @@ Create unified labels for cinder-csi components
 {{- define "cinder-csi.common.matchLabels" -}}
 app: {{ template "cinder-csi.name" . }}
 release: {{ .Release.Name }}
+{{- if .Values.extraLabels }}
```

Comment:
Hm I wonder if this shouldn't go in `cinder-csi.common.metaLabels` instead? Of course it depends on what exactly we want to label, but https://github.com/kubernetes/cloud-provider-openstack/pull/1788 adds `extraLabels` into `metadata.labels`, not `spec.selector.matchLabels`. What do @ramineni @jichenjc think?

Reply:
I reckon you're right @gman0. Happy to move it to that template if you all agree

Reply:
+1 it should be in metaLabels